.github/workflows/cc_kbc.yml

```diff
@@ -46,11 +49,11 @@ jobs:
 
       - name: Build and install with cc_kbc feature
```

Comment:
maybe change this to:

```
name: Build and install with ${{ matrix.kbc }} feature
```

(assuming that you can use these interpolations in the `name` field; not sure)